applications/ila-cookie-banner/ila-cookie-banner.html

```diff
@@ -0,0 +1,194 @@
+<!DOCTYPE html>
+
+<html lang="en" class="il-toolkit">
+
+<head>
+    <meta charset="utf-8" />
+    <title>ILA CookieB</title>
+
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="dns-prefetch" href="https://cdn.toolkit.illinois.edu" />
+    <link rel="icon" href="https://cdn.brand.illinois.edu/favicon.ico" />
+    <link type="text/css" rel="stylesheet" href="https://cdn.toolkit.illinois.edu/2.16/toolkit.css" />
+    <script src="https://cdn.toolkit.illinois.edu/2.16/toolkit.js"></script>
+    <title>Cookie Banner Example</title>
+    <link rel="stylesheet" href="ila-cookie-banner.css" />
+    <link rel="stylesheet" href="../ila-slideovers/ila-slideover.css" />
+    <style>
+        body {
+            font-weight: 400;
+            font-size: 18px;
+            line-height: 1.6666666667em;
+            font-family: var(--il-font-sans);
+            color: var(--il-text-color);
+        }
+
+        a {
+            transition: color 0.3s;
+            color: var(--il-link-color);
+            text-decoration: underline;
+        }
+
+        a:focus,
+        a:hover {
+            outline-style: none;
+            color: var(--il-focused-link-color);
+        }
+
+        .button-holder {
+            display: flex;
+            position: relative;
+            z-index: 10;
+        }
+
+        .button-holder button {
+            margin-right: 5px;
+        }
+    </style>
+</head>
+
+<body onload="openCookieB('ilaCookieBOne')">
+    <il-page>
+        <div class="cookieb-holder" role="alert">
+            <div class="cookieb-holder-inner">
+                <div id="ilaCookieBOne" class="ila-cookieb ila-cookieb--closed">
+                    <div class="ila-cookieb__cookieb">
+                        <button type="button" class="ila-cookieb__close-button" aria-label="close cookie banner"
+                            onclick="closeCookieB('ilaCookieBOne')">
+                            <div class="ila-cookieb__close-icon il-icon" role="presentation">
+                                cancel
+                            </div>
+                        </button>
+                        <div class="ila-cookieb__heading">
+                            <h3 class="ila-cookieb__h3">Cookie Notice</h3>
```

Comment:
May cause semantic ordering issue having an H3 without context.

Reply:
We corrected this to H2 this morning. Thanks!